lib/tasks/handover/handover_emails.rake

```diff
@@ -4,4 +4,16 @@ namespace :handover do
     Rails.logger = Logger.new($stdout)
     Handover::HandoverEmailBatchRun.send_all_upcoming_handover_window
   end
+
+  desc 'send_all_handover_date'
```

Comment:
These 2 descs could be more descriptive - optional

Reply:
I shall describify them!